client/utils.ts

```diff
@@ -8,6 +8,14 @@ export function stringifyAirport(airport: Airport|null) {
     return (airport.iata || airport.icao) + " - " + airport.municipality + "/" + airport.country;
 }
 
+export function stringifyAirline(airline: Airline|null) {
+    if(airline === null) {
+        return "N/A";
+    }
+
+    return (airline.name + " - " + airline.icao);
```

Comment:
Just for the sake of consistency with `stringifyAirport`, I would stringify an airline as `airline.icao + " - " + airline.name` instead.